qaframework-bdd-tests/pom.xml

```diff
@@ -47,7 +47,7 @@
 		<dependency>
 			<groupId>org.seleniumhq.selenium</groupId>
 			<artifactId>selenium-java</artifactId>
-			<version>4.3.0</version>
+			<version>4.1.3</version>
```

Comment:
Any Reason of reverting it back ?

Reply:
Am just testing out something but will get it back to the latest version.

Reply:
Any version higher than 4.1.4 breaks all the tests so have instead upgraded to 4.1.4